lib/rules/max-line-length/__tests__/index.js

```diff
@@ -238,7 +238,7 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
-  config: [ 20, { ignorePattern: "/https?:\/\/[0-9,a-z]*.*/" } ],
+  config: [ 20, { ignorePattern: "/https?://[0-9,a-z]*.*/" } ],
```

Comment:
@Arcanemagus I hope you don't mind me pinging you, but do you have any thoughts on this? You seemed knowledgeable about this subject in the other issue: https://github.com/stylelint/stylelint/issues/2673.